scapy/fields.py

```diff
@@ -1901,38 +1912,40 @@ def __init__(
             length_of=None,  # type: Optional[str]
             fmt="H",  # type: str
             count_of=None,  # type: Optional[str]
-            adjust=lambda pkt, x: x,  # type: Callable[[BasePacket, int], int]
-            fld=None,  # type: Optional[Any]
+            adjust=lambda pkt, x: x,  # type: Callable[[Packet, int], int]
     ):
         # type: (...) -> None
         Field.__init__(self, name, default, fmt)
         self.length_of = length_of
         self.count_of = count_of
         self.adjust = adjust
-        if fld is not None:
```

Comment:
Why are you removing this part?

Reply:
This is unused. I'm assuming this is an alias that got deprecated long ago.